pyspecProcScripts/load_data.py

```diff
@@ -349,28 +349,9 @@ def proc_Hahn_echoph(s, fl=None):
 def proc_spincore_IR(s, fl=None):
     if "t" in s.dimlabels:
         s.chunk("t", ["ph2", "ph1", "t2"], [2, 2, -1])
-    s.setaxis("ph1", r_[0, 2.0] / 4)
-    s.setaxis("ph2", r_[0, 2.0] / 4)
-    s.reorder(["ph1", "ph2"]).set_units("t2", "s")
-    s.set_prop("coherence_pathway", {"ph1": 0, "ph2": +1})
-    s.set_units("t2", "s")
-    s["t2"] -= s.get_prop("acq_params")["tau_us"] * 1e-6
-    s *= s.shape["nScans"]
-    s.squeeze()
-    s.ft("t2", shift=True)
-    s.ft(["ph1", "ph2"])
-    if fl is not None:
-        fl.next("raw data -- coherence channels")
-        fl.image(s.C.setaxis("vd", "#").set_units("vd", "scan #"))
-    s.ift("t2")
-    if fl is not None:
-        fl.next("time domain (all $\\Delta p$)")
-        fl.image(s.C.setaxis("vd", "#").set_units("vd", "scan #"))
-    s.ft("t2")
-    if fl is not None:
-        fl.next("frequency domain (all $\\Delta p$)")
-        fl.image(s.C.setaxis("vd", "#").set_units("vd", "scan #"), black=False)
-    return s
+    if s.get_prop("coherence_pathway") is None:
+        s.set_prop("coherence_pathway", {"ph1": 0, "ph2": +1})
```

Comment:
here it seemed it might be wise to add info that was missing before, but I don't see anything at a glance in the stuff that I'm deleting that's not handled by the generalproc function